tests/test_yokogawa/test_yokogawa_6370.py

```diff
@@ -35,6 +36,88 @@ def test_init():
         pass
 
 
+def test_id():
+    with expected_protocol(
+        ik.yokogawa.Yokogawa6370,
+        [":FORMat:DATA REAL,64", "*IDN?"],
+        ["'YOKOGAWA,AQ6370D,x,02.08'"],
+    ) as inst:
+        assert inst.id == "YOKOGAWA,AQ6370D,x,02.08"
+
+
+def test_status():
+    with expected_protocol(
+        ik.yokogawa.Yokogawa6370, [":FORMat:DATA REAL,64", "*STB?"], ["7"]
+    ) as inst:
+        assert inst.status == 7
+
+
+def test_operation_event():
+    with expected_protocol(
+        ik.yokogawa.Yokogawa6370,
+        [":FORMat:DATA REAL,64", ":status:operation:event?"],
+        ["7"],
+    ) as inst:
+        assert inst.operation_event == 7
+
+
+@given(
+    values=st.lists(st.floats(allow_infinity=False, allow_nan=False), min_size=1),
+    channel=st.sampled_from(ik.yokogawa.Yokogawa6370.Traces),
+    axis=st.sampled_from(["X", "Y"]),
```

Comment:
Here it might be better to use a `@pytest.mark.parametrize` fixture to just check both axes. No need to sample via `hypothesis`

Reply:
Ok, I never worked with hypothesis. Thanks for the hint.